app/src/components/market/sections/market_profile/market_status/closed.tsx

```diff
@@ -335,14 +336,25 @@ const Wrapper = (props: Props) => {
     ? balances.reduce((acc, balance, index) => (payouts[index].gt(0) ? acc.add(balance.shares) : acc), new BigNumber(0))
     : new BigNumber(0)
   const EPS = 0.01
-  const allPayoutsEqual = payouts
-    ? payouts.every(payout =>
-        payout
-          .sub(1 / payouts.length)
-          .abs()
-          .lte(EPS),
-      )
-    : false
+
+  let invalid = false
+
+  if (isScalar) {
+    if (question.answers && question.answers[question.answers.length - 1].answer === INVALID_ANSWER_ID) {
```

Comment:
Should this be if (!question.answers || (question.answers && question.answers[question.answers.length - 1].answer === INVALID_ANSWER_ID)) {

}

Reply:
No, because a market has to be explicitly answered as invalid to be invalid